airflow/providers/amazon/aws/utils/__init__.py

```diff
@@ -21,13 +21,17 @@
 from datetime import datetime
 from enum import Enum
 
+from deprecated import deprecated
+
+from airflow.utils.helpers import prune_dict
 from airflow.version import version
 
 log = logging.getLogger(__name__)
 
 
+@deprecated(reason="use prune_dict() instead")
```

Comment:
nit: maybe be more specific?
```suggestion
@deprecated(reason="use airflow.utils.helpers.prune_dict() instead")
```